index.rst

```diff
@@ -189,6 +189,8 @@ Guide for contributing to Python:
 * Advanced tasks for once you are comfortable
     * :doc:`silencewarnings`
     * Fixing issues found by the :doc:`buildbots <buildbots>`
+    * Helping out with `Open Reviews`_.
```

Comment:
"Open Reviews" might not immediately make sense to people. Perhaps say 'Helping out with reviewing open pull requests'?